openvino/README.md

```diff
@@ -34,6 +34,28 @@ Clear Linux* OS based container images use:
 
 To learn more about Clear Linux* OS, visit: https://clearlinux.org.
 
+## Supported Devices
+
+The dldt package in Clear Linux enables support for below devices.
+
+
+| PLUGIN               | DEVICE TYPES |
+| ---------------------| -------------|
+| CPU plugin           | Intel® Xeon® with Intel® AVX2 and AVX512, Intel® Core™ Processors with Intel® AVX2, Intel® Atom® Processors with Intel® SSE |
```

Comment:
default CPU plugin is enabled on Clear container, right?

Reply:
sure

Reply:
Does Clear container support CPU extensions library? 
https://docs.openvinotoolkit.org/2019_R1/_inference_engine_src_extension_README.html

Reply:
From 2020.1 release, CPU extensions library was removed. Extensions were moved into the plugin. In current clearlinux/openvino container (2019.R3.1), CPU extensions library is "libcpu_extension.so" which is in the dldt package.

Reply:
good. LGTM.